ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -513,40 +513,40 @@ object Config {
           )
 
         // TODO append-only: remove after removing support for the current (mutating) schema
-        opt[Unit]("index-append-only-schema-unsafe")
+        opt[Unit]("index-append-only-schema")
           .optional()
           .hidden()
           .text(
-            s"Use the append-only index database with parallel ingestion. Highly unstable. Should not be used in production."
+            s"Use the append-only index database with parallel ingestion."
```

Comment:
```suggestion
            "Use the append-only index database with parallel ingestion."
```

Reply:
Sorry, the PR went in on automerge before I was able to react